lib/contentful.js

```diff
@@ -142,46 +151,54 @@ module.exports.fetchCampaignIdsWithKeywords = function () {
 module.exports.fetchKeyword = function (keyword) {
   logger.debug(`contentful.fetchKeyword:${keyword}`);
 
-  return exports.fetchSingleEntry({
-    content_type: 'keyword',
-    'fields.keyword': keyword,
-  });
+  const query = exports.getQueryBuilder()
+    .type('keyword')
+    .keyword(keyword)
+    .build();
+  return exports.fetchSingleEntry(query);
+};
+
+module.exports.fetchKeywords = function fetchKeywords() {
+  logger.debug('contentful.fetchKeywords');
+  const query = exports.getQueryBuilder()
+    .type('keyword')
+    // We store keywords for both Thor and Production as entries, so filter by current environment.
+    .env(process.env.NODE_ENV)
+    .build();
+  return exports.getClient().getEntries(query)
+  .then(response => response.items.map(item => exports.formatKeywordFromResponse(item)))
+  .catch(error => exports.contentfulError(error));
 };
 
 module.exports.fetchKeywordsForCampaignId = function (campaignId) {
   logger.debug(`contentful.fetchKeywordsForCampaignId:${campaignId}`);
 
-  // TODO: refactor into a function that creates the query.
-  const query = {
-    content_type: 'keyword',
+  const query = exports.getQueryBuilder()
+    .type('keyword')
     // We store keywords for both Thor and Production as entries, so filter by current environment.
-    'fields.environment': process.env.NODE_ENV,
-  };
-  if (campaignId) {
-    query['fields.campaign.sys.contentType.sys.id'] = 'campaign';
-    query['fields.campaign.fields.campaignId'] = campaignId;
-  }
-
+    .env(process.env.NODE_ENV)
+    .custom({
+      'fields.campaign.sys.contentType.sys.id': 'campaign',
+      'fields.campaign.fields.campaignId': campaignId,
```

Comment:
As far as the naming convention, if it adds clarity to the method to call it `campaignId` vs `campaign` then yes. To my brain, having a `campaignId` method, sort of lets me wonder if there is another way to add more things about the campaign or field, like `campaignName`, `contentTypeId`, etc. Calling it campaign, adds the benefit to override it to accept an object and send more specific data like `.campaign({id: 1, closed: false})`, which would be weird if it was named `.campaignId({id:1, closed: true})`. This is the reason I named it this way, but I'm totes open to what you guys think.